scripts/add_blacklist_guids.py

```diff
@@ -0,0 +1,24 @@
+from framework.mongo import database as db
+from website.app import init_app
+
+blacklist_file = 'guid_blacklist.txt'
```

Comment:
In order to generate the guids in `guid_blacklist.txt` you need files that contain bad/inappropriate words...probably best not to have that in the osf so that it doesn't accidentally get committed by anyone.